### ISSUE TITLE
In markdown files, hitting enter in a list item always starts a new list item

### ISSUE THREAD

**Issue by dead10ck** [mawww/kakoune#4197]
### Feature 

I'm not sure whether to classify this as a bug or not, as I'm guessing it was intentional. When writing markdown, if you are writing a list, and you hit the enter key, it always inserts a new bullet. This makes it really annoying when writing lists, since more often than not, list items span multiple lines, and this behavior means that every time I hit enter, I have to backspace to delete the new bullet that was inserted. It would be better if it just continued the line on the same indentation level. If I want to insert a new bullet item, I will backspace and insert a `*` myself. This is how vim works.

**Comment by krobelus** [comment]
I agree that it's often not helpful.
You can disable this behavior with `set global disabled_hooks markdown-insert` or `set global disabled_hooks .*-insert`

**Comment by dead10ck** [comment]
@krobelus yeah, but then it doesn't even go to the same indentation level, which is just as bad.
